### ISSUE TITLE
[Android] Player Progress not getting Proper

### ISSUE THREAD

**Issue by nikesh8** [doublesymmetry/react-native-track-player#1332]
I am using the latest version of `react-native-track-player` from that I am using the `useProgress()` hook for showing player progress but on android, I am getting progress seconds in the manner of 2,4,6,8, etc. It is working fine for iOS apps. So, Can anyone help me out with this issue?

Here's the Package version:
```
"react": "17.0.2",
"react-native": "0.66.1",
"react-native-track-player": "2.1.2",
```

Here's the sample code for my Progress component:
```
const ProgressBar = () => {
  const progress = useProgress();
  const [seeking, setSeeking] = useState(false);

  const seekTo = time => {
    TrackPlayer.seekTo(time);
    logAudioSeek(time);
    setSeeking(false);
  };

  const slidingStart = () => {
    setSeeking(true);
  };

  const position = formatTime(Math.floor(progress.position));
  const diff = Math.floor(progress.duration - progress.position);
  const duration = formatTime(
    Math.floor(progress.duration) < 0 ? 0 : diff > 0 ? diff : 0,
  );
  const sliderPosition =
    Math.floor(progress.position) < 0 ? 0 : Math.floor(progress.position);
  const sliderMaxValue =
    Math.floor(progress.duration) < 0 ? 0 : Math.floor(progress.duration);
  return (
    <View style={styles.mainContainer}>
      <View style={styles.sliderContainer}>
        <Slider
          minimumValue={0}
          maximumValue={sliderMaxValue}
          step={1}
          value={sliderPosition}
          style={styles.container}
          trackStyle={styles.track}
          thumbStyle={[styles.thumb, seeking && styles.seekingThumb]}
          minimumTrackTintColor={Colors.white}
          thumbTouchSize={{ width: 50, height: 40 }}
          onSlidingComplete={value => seekTo(value)}
          onSlidingStart={slidingStart}
        />
        <View
          style={{
            flexDirection: 'row',
            justifyContent: 'space-between',
            alignItems: 'flex-end',
            height: 20,
          }}>
          <View style={styles.positionContainer}>
            <Text style={styles.position}>{position}</Text>
          </View>
          <View style={styles.durationContainer}>
            <Text style={styles.duration}>
              {Math.floor(progress.duration) > 0 && '-'}
              {duration}
            </Text>
          </View>
        </View>
      </View>
    </View>
  );
};
```

Thanks in advance :)

**Comment by krunalbadamicappital** [comment]
I am also facing this issue 😔

**Comment by bradleyflood** [comment]
Thanks @nikesh8 @krunalbadamicappital are you able to investigate and provide a PR or fix? We would love some help.

Thank you

**Comment by bradleyflood** [comment]
Also see #1371

**Comment by jspizziri** [comment]
@nikesh8 can you test v2.2.0-rc3 and see if this is still an issue?

If it is still an issue in that version then I'll get it added to the issues to fix for `v2.2.0`. However, if it's not an issue in that version then you'll need to upgrade.

**Comment by jspizziri** [comment]
@nikesh8 , I'm closing this issue for the following reasons:

1. It's not actually clear to me what your issue is.
2. The example app seems to be working as I'd expect. [Please take a look at that progress bar for inspiration.](https://github.com/doublesymmetry/react-native-track-player/blob/main/example/src/components/Progress.tsx)
3. I don't see any issues with the most recent version it's currently working on Android.
4. Lack of response.

Best of luck!
